test/controllers/journeys/adjustments/profitOrLoss/CurrentYearLossControllerCarryLossForwardSpec.scala

```diff
@@ -42,7 +45,7 @@ class CurrentYearLossControllerCarryLossForwardSpec extends BooleanGetAndPostQue
     view(expectedForm, taxYear, businessId, scenario.userType, scenario.mode).toString()
   }
 
-  // TODO in SASS-9566 test that the view CarryLossForward gets displayed when user has NO other incomes
-}
```

Comment:
Should a comment remain to say to test for when user has no otherIncomeSources but has pensionIncomes (to match the change in the controller)?